### ISSUE TITLE
entities showing up in cut text

### ISSUE THREAD

**Issue by MikeTheVike** [BlackBeltDesigns/bbd_chopper.ee_addon#4]
Using BBD Chopper to limit the words to 25 on entries. There is an apostrophe in the text that uses **"&rsquo;"** in the html. This appears in the text on the front end. is there a way to make this work with entities? Thanks!
![screen shot 2017-05-01 at 9 24 27 am](https://cloud.githubusercontent.com/assets/220292/25581890/0260ff90-2e50-11e7-8d48-e8940210b185.png)

**Comment by BlackBeltDesigns** [comment]
Mike,

Can you give me a snippet of text you are trying to limit and the code you used? This way I can test against exactly what you are after. 

Thanks.

**Comment by MikeTheVike** [comment]
Sure, I'll also attach some screenshots in case something doesn't come over correctly. Wygwam is the fieldtype. Thanks for looking into this and let me know if you need anything else!
The template code is...
                <p>
                {exp:chopper words="25" append="&hellip;"}
                    {blog_content}
                {/exp:chopper}
                </p>

The text that gets cut off has an apostrophe  in the word "car's"...

Never leave a pet unattended in a parked car when the temperature is more than 70 degrees!
When it’s 72 degrees outside, a car’s temperature can rocket to 116 degrees





Mike Lohrman
On Monday, May 1, 2017, 10:29:15 AM CDT, BBDOKC <notifications@github.com> wrote:
Mike,

Can you give me a snippet of text you are trying to limit and the code you used? This way I can test against exactly what you are after.

Thanks.

**Comment by BlackBeltDesigns** [comment]
I used your copy in a RTE and in a wygwam field. Both worked properly. (See images)
This was taking your copy and pasting it as plain text. If you use a general paste, you will carry over HTML entities. That's what it looks like is happening. 

If you absolutely NEED HTML entities, let me know and I can look at rewriting things to allow for those.  ;)

![code](https://cloud.githubusercontent.com/assets/987074/25593418/119f61e0-2e82-11e7-939e-5e1e4dc0d6e9.PNG)
![output](https://cloud.githubusercontent.com/assets/987074/25593417/119c9d66-2e82-11e7-9b6e-3ee02609e8ce.PNG)

**Comment by MikeTheVike** [comment]
Thank you, definitely not an emergency. I think I did paste as plain text, but don't remember. I don't think this will be an issue the majority of the time. Thanks again.
On Monday, May 1, 2017, 3:29:22 PM CDT, BBDOKC <notifications@github.com> wrote:
I used your copy in a RTE and in a wygwam field. Both worked properly. (See images)
This was taking your copy and pasting it as plain text. If you use a general paste, you will carry over HTML entities. That's what it looks like is happening.

If you absolutely NEED HTML entities, let me know and I can look at rewriting things to allow for those. ;)
